package.json

```diff
@@ -82,16 +82,18 @@
   ],
   "jspm": {
     "main": "js/bootstrap",
+    "ignore": ["dist/js/npm"],
+    "directories": {
+      "lib": "dist"
+    },
     "shim": {
       "js/bootstrap": {
-        "deps": "jquery",
+        "deps": ["jquery"],
         "exports": "$"
       }
     },
-    "files": [
-      "css",
-      "fonts",
-      "js"
-    ]
+    "dependencies": {
+      "jquery": "github:components/jquery"
```

Comment:
Sure, unfortunately we don't currently support arbitrary version ranges (just semver and tilde), so it may be best to stick with `*` for jQuery in this case.